lib/ansible/module_utils/network/ios/utils/utils.py

```diff
@@ -30,7 +30,7 @@ def add_command_to_config_list(interface, cmd, commands):
 
 def dict_to_set(sample_dict):
     # Generate a set with passed dictionary for comparison
-    test_dict = {}
+    test_dict = dict()
```

Comment:
Just a note: these days ```{}``` is preferred to ```dict()```.  (literals are faster than function calls and, especially for empty dicts, the literal is more idiomatic)